account_move_batch_validate/wizard/account_move_validate.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2014 Camptocamp SA, 2017 ACSONE
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo import api, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+try:
+    from odoo.addons.queue_job.job import job
+except ImportError:
+    _logger.debug('Can not `import queue_job`.')
+    import functools
+
+    def empty_decorator_factory(*argv, **kwargs):
+        return functools.partial
+    job = empty_decorator_factory
+
+
+class AccountMoveValidate(models.TransientModel):
+
+    _inherit = 'validate.account.move'
+
+    action = fields.Selection(
+        selection='_get_actions', string="Action",
+        required=True, default='mark')
+    eta = fields.Integer(string="Seconds to wait before starting the jobs")
+    asynchronous = fields.Boolean(string="Use asynchronous validation")
+
+    @api.model
+    def _get_actions(self):
+        return [
+            ('mark', 'Mark for posting'),
+            ('unmark', 'Unmark for posting')
+        ]
```

Comment:
Actions are not translatable like this. Move them directly on field definition (and remove function).

Reply:
thanks, I think explicit marking them as translatable will solve the issue